data/SIMS Activation Log.csv

```diff
@@ -0,0 +1,34 @@
+Year,Country,Code,Response,Support,Significance (1-5; 5 as most significant activations),Start Date,End Date
```

Comment:
This should *not* be a static file on the website it will be out of date very quickly. Make this an AJAX call to a Google Sheet. Who is tasked with maintaining this? How are the severities decided?